src/main/java/com/starkinfra/IssuingCard.java

```diff
@@ -0,0 +1,717 @@
+    - before [datetime.date or string, default None] date filter for objects created only before specified date. ex: datetime.date(2020, 3, 10)
+    - tags [list of strings, default None]: tags to filter retrieved objects. ex: ["tony", "stark"]
+    - ids [list of strings, default None]: list of ids to filter retrieved objects. ex: ["5656565656565656", "4545454545454545"]
+    - limit [integer, default None]: maximum number of objects to be retrieved. Unlimited if None. ex: 35
+    - expand [list of strings, default []]: fields to expand information. ex: ["rules", "security_code", "number", "expiration"]
+     * <p>
+     * Return:
+     * @return generator of IssuingCards objects with updated attributes
+     * @throws Exception error in the request
+     */
+    public static Generator<IssuingCard> query(Map<String, Object> params) throws Exception{
+        return Rest.getStream(data, params, null);
+    }
+
+    /**
+     * Retrieve IssuingCards
+     * <p>
+     * Receive a generator of IssuingCard objects previously registered in the Stark Bank API
+     * <p>
+     * Parameters:
+     * @param user [Project object, default null]: Project object. Not necessary if StarkBank.Settings.user was set before function call
+     * <p>
+     * Return:
+     * @return generator of IssuingCards objects with updated attributes
+     * @throws Exception error in the request
+     */
+    public static Generator<IssuingCard> query(User user) throws Exception{
+        return Rest.getStream(data,  new HashMap<>(), user);
+    }
+
+
+    /**
+     * Retrieve IssuingCards
+     * <p>
+     * Receive a generator of IssuingCard objects previously registered in the Stark Bank API
+     * <p>
+     * Return:
+     * @return generator of IssuingCards objects with updated attributes
+     * @throws Exception error in the request
+     */
+    public static Generator<IssuingCard> query() throws Exception{
+        return Rest.getStream(data, new HashMap<>(), null);
+    }
+
+    public final static class Page {
+        public List<IssuingCard> cards;
+        public String cursor;
+
+        public Page(List<IssuingCard> cards, String cursor) {
+            this.cards = cards;
+            this.cursor = cursor;
+        }
+    }
+
+    /**
+     * Retrieve paged IssuingCards
+     * <p>
+     * Receive a list of up to 100 IssuingCard objects previously registered in the Stark Bank API and the cursor to the next page.
+     * <p>
+     * Parameters:
+     * @param params map of parameters
+     * cursor [string, default None]: cursor returned on the previous page function call
+     * limit [integer, default None]: maximum number of objects to be retrieved. Unlimited if None. ex: 35
+     * @param user [Project object, default null]: Project object. Not necessary if StarkBank.Settings.user was set before function call
+     * <p>
+     * Return:
+     * @return  list of IssuingCard objects with updated attributes

```

Comment:
* Invoice.Page.issuingCards: list of IssuingCards objects with updated attributes
     * Invoice.Page.cursor: cursor to retrieve the next page of IssuingCards objects